gatherers/manifest.js

```diff
@@ -35,8 +35,8 @@ class Manifest extends Gather {
       const finalURL = url.resolve(options.url, manifestURL);
 
       request(finalURL, function(err, response, body) {
```

Comment:
hmm, what do you mean? We don't have any gatherers tests yet, and by the time it gets up to audits it's just a `debugString`, which we do test for correctly passing through.

Another twist on this is this will all change when we switch to letting the page fetch the manifest in #83 (which I'm working on next)